tests/functional/object/rgw/test_rgw_sts.py

```
@@ -0,0 +1,534 @@
+        ).get()["metadata"]["name"]
+        toolbox = get_ceph_tools_pod()
+
+        # Generate unique IDs to avoid conflicts
+        # RGW account ID must be exactly 20 bytes long, start with "RGW", and end with numeric digits
+        # Format: RGW + 17 numeric digits = 20 bytes total
+        numeric_suffix = str(uuid.uuid4().int)[:17]
+        account_id = f"RGW{numeric_suffix}"
+        user_id = create_unique_resource_name("sts", "user")
+
+        # Step 1: Create RGW account
+        logger.info(f"Creating RGW account with ID: {account_id}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)